x-pack/plugins/cases/public/components/configure_cases/index.tsx

```diff
@@ -74,7 +76,7 @@ const ConfigureCasesComponent: React.FC<ConfigureCasesProps> = ({ userCanCrud })
     refetchCaseConfigure,
     setConnector,
     setClosureType,
-  } = useCaseConfigure();
+  } = useCaseConfigure(owner[0]);
```

Comment:
Is there any chance `owner` could be empty? Or is it guaranteed to be populated with at least one value as part of the react context?

Reply:
You are right about it. I had that debate with myself. After some thought, I decided to do it as [`9fc4772` (#99478)](https://github.com/elastic/kibana/pull/99478/commits/9fc4772b9c2d8e2b354dbb8a92be500fb5dbeaf9). If the array is empty when using the context an error will be thrown.

Reply:
When cases have their own UI then we should reconsider what it means to have an empty owner.